docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,111 @@
+---
+title: The newest additions to MUI X v6
+description: Support for time zones, a new filtering UI, charts! and more
+date: 2023-07-30T00:00:00.000Z
+authors: ['josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/card.png" alt="" style="margin-bottom: 16px;" width="2400" height="559" />
+
+Since the first v6 stable release, we have continuously rolled out new major features.
+Now that we reached [MUI X v6.10.0](https://github.com/mui/mui-x/releases/tag/v6.10.0), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Date Pickers](#date-pickers)
+  - [Support for Time Zones](#support-for-time-zones)
+  - [Digital Clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and Paste](#copy-and-paste)[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+  - [Filter on Column Headers](#filter-on-column-headers)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+  - [Performance Enhancements: Smoother Scrolling and Expanding Rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
+- [Charts - Alpha Version](#charts---alpha-version)
+- [What's next](#whats-next)
+
+## Date pickers
+
+### Support for time zones
+
+No more hassle displaying dates on the time zones of your choice. 🌎
+
+You can effortlessly display and select dates and times in different time zones, without worrying about the conversion logic and keeping the stored value consistent with your backend.
+
+**[live demo showcasing stored date and toggle between system/UTC/timezone]**
+
+Use adapters to control the original value and set the `timezone` prop to define in which timezone the Date, Time, or DateTime should be displayed.
+
+```tsx
+<TimePicker
+  value={value} // Can be in any timezone of your choice
+  onChange={setValue}
+  timezone="Pacific/Honolulu"
+  label={'Rendered in "Pacific/Honolulu"'}
+/>
+```
+
+Please check the [full instructions](https://deploy-preview-8261--material-ui-x.netlify.app/x/react-date-pickers/timezone/) for more information on using this feature in your application.
+
+### Digital clock
+
+This feature is an alternative to the analog clock on time pickers; the original proposal was mainly focused on mobile devices, whereas the new interface is designed for a desktop experience or better time precision. This new addition makes our date pickers even more versatile and user-friendly and is available as the default variant for desktops on both [TimePicker](https://mui.com/x/react-date-pickers/time-picker/) and [DateTimePicker](https://mui.com/x/react-date-pickers/date-time-picker/).
```

Comment:
I feel like this is not needed. No need to talk about original proposal, especially if it was not implemented.

Reply:
Ahh, Highlighting didnt work. I only wanted to review this portion:

`the original proposal was mainly focused on mobile devices, whereas the new interface is designed for a desktop experience or better time precision.`

Reply:
Thanks for pointing this out. Would you have the same perspective if the original proposal was implemented and still is a possible alternative to use? How can we better write that to make this point?

Reply:
oh, okay. The original text made it seem like the analog Time Clock will soon be replaced by the Digital Clock because the limitation of desktop utilization.

 
Totally up to you, I would write something like: 

'While the original proposal for an analog Time Clock for time pickers focused primarily on mobile devices, the newly introduced Digital Clock caters to desktop users and offers improved time precision. This addition complements the existing alternative, making our date pickers even more versatile and user-friendly. As a result, developers have the option to choose between the Time Clock or the Digital Clock, both available as default variants for desktops on TimePicker and DateTimePicker.'

(I am open to both variation. Feel free to close this conversation)